Makefile

```diff
@@ -189,6 +189,7 @@ PRODUCT_NAME = etcher
 APPLICATION_NAME_LOWERCASE = $(shell echo $(APPLICATION_NAME) | tr A-Z a-z)
 APPLICATION_VERSION_DEBIAN = $(shell echo $(APPLICATION_VERSION) | tr "-" "~")
 APPLICATION_VERSION_REDHAT = $(shell echo $(APPLICATION_VERSION) | tr "-" "~")
+PACKAGE_TYPE_PROPERTY = packageType
```

Comment:
I'm in two minds whether it's worth having a separate variable for this - it looks a bit weird having both:
```
--extraMetadata.version=$(APPLICATION_VERSION) \
--extraMetadata.$(PACKAGE_TYPE_PROPERTY)=dmg
```
and I wonder if it'd look better / more consistent as:
```
--extraMetadata.version=$(APPLICATION_VERSION) \
--extraMetadata.packageType=dmg
```
instead :question: